a10_neutron_lbaas/vthunder/instance_manager.py

```diff
@@ -377,6 +380,28 @@ def plumb_instance_subnet(self, instance_id, subnet_id, allowed_ips, wrong_ips=[
         network_id = subnet["subnet"]["network_id"]
         return self.plumb_instance(instance_id, network_id, allowed_ips, wrong_ips=wrong_ips)
 
+    def build_server_with_defaults(self, server, vthunder_config):
+        copy_keys = [("image", "glance_image"),
+                     ("flavor", "nova_flavor"),
+                     ]
+
+        for server_key, config_key in copy_keys:
+            # if the server doesn't have this attribute configured, set it from default
+            # Sentinel is the value set by the API when it's unspecified.
+            if server_key not in server or type(server[server_key]) == neutron_const.Sentinel:
+                server[server_key] = vthunder_config.get(config_key)
```

Comment:
This is what the `resources.remove_attributes_not_specified` call does in the other plugins